### DISCUSSION TITLE
Automate documentation and generate PDF

### DISCUSSION THREAD

**Discussion by almenscorner** [almenscorner/IntuneCD#42] [Show and tell]
Hi all 👋

If you are using this tool to backup and document your Intune environment and been thinking you want the documentation as a PDF file instead of Markdown to give to the people that don't understand how to read .md files, then @aaronparker has written two excellent posts describing how to do this with both GitHub actions and Azure DevOps pipelines. Check them out,

https://stealthpuppy.com/automate-intune-documentation-azure/
https://stealthpuppy.com/automate-intune-documentation-github/

Happy automating 😎

**Comment by aaronparker** [comment]
I've updated a couple of template repositories for both GitHub and Azure DevOps - the workflows/pipelines produce the as-built in both PDF and HTML output with CSS formatting:

https://github.com/aaronparker/intune-backup-template
https://github.com/aaronparker/intune-backup-azure
